src/Header/Header.jsx

```diff
@@ -1,20 +1,64 @@
 import React, { useContext } from 'react';
-import { Layout } from 'antd';
+import PropTypes from 'prop-types';
+import { Col, Icon, Layout, Row } from 'antd';
+import { emptyFn } from '../helper';
 import { ThemeContext } from '..';
 
 const { Header: AntdHeader } = Layout;
 
 export const Header = (props) => {
```

Comment:
Wenn die Navigation oder die Extras zu viel Inhalt haben, dann passen die Breakpoints nicht mehr. Deshalb habe ich das nicht in den Header eingebaut. In dem Fall mag das optimiert sein auf unseren Inhalt, aber in anderen Tools die wir damit machen werden, ist die Wahrscheinlichkeit hoch das es nicht mehr passt. Daher habe ich das als Children übergeben gehabt, damit man die Breakpoints stets optimieren kann. Kann man zwar jetzt immer noch über die children machen aber dann bringt uns die Änderung in dem Header jetzt nur in einem sehr begrenzten Maße was

Reply:
Habe ich angepasst. Die Breakpoints für Menu und Extra lassen sich jetzt per Props konfigurieren